### PR TITLE
Update Submodules

### PR DESCRIPTION
**Pull request by sbSteveK**
```
aws-c-auth         v0.7.22 -> v0.7.25
aws-c-cal          v0.7.1 -> v0.7.3
aws-c-common       v0.9.23 -> v0.9.27
aws-c-event-stream v0.4.2 -> v0.4.3
aws-c-http         v0.8.4 -> v0.8.8
aws-c-io           v0.14.11 -> v0.14.18
aws-c-s3           v0.6.2 -> v0.6.4
aws-c-sdkutils     v0.1.16 -> v0.1.19
aws-lc             v1.32.0 -> v1.33.0
s2n                v1.4.17 -> v1.5.0
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/test_websocket.py [hunk 1]
```diff
@@ -484,10 +484,13 @@ def bad_completion_callback(data):
             # wait for shutdown...
             handler.shutdown_future.result(TIMEOUT)
 
+            # Change in this PR in aws-c-io https://github.com/awslabs/aws-c-io/pull/650
+            # along with this PR in aws-c-http https://github.com/awslabs/aws-c-http/pull/474
+            # allows a window update during shutdown.
```
Comment (TingDaoK) [inline test/test_websocket.py:489]: hmmm, how is this related to window update during shutdown? It seems sending data after shutdown to me?

Comment (sbSteveK) [reply test/test_websocket.py:489]: Restoring test thanks to https://github.com/awslabs/aws-c-http/pull/483#pullrequestreview-2240952701

**Review by graebm — APPROVED** [review]
